psp/scripts/eval_model.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import datetime
+import pathlib
+
+# import altair as alt
+import numpy as np
+import pandas as pd
+import tqdm
+
+from psp.data.data_sources.pv import PvDataSource
+from psp.ml.metrics import MeanRelativeError, Metric, mean_absolute_error
+from psp.ml.models.previous_day import PreviousDayPvSiteModel
+from psp.ml.training import make_data_loader
+
+METRICS: dict[str, Metric] = {
+    "mre_cap=1": MeanRelativeError(cap=1),
+    "mae": mean_absolute_error,
+}
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(__doc__)
+    parser.add_argument("-p", "--pv-data", required=True, help=".netcdf PV data")
+    parser.add_argument(
+        "-n",
+        "--exp-name",
+        help="name of the experiment (used as the name of the dictionary)",
+    )
+    parser.add_argument("-s", "--sample", type=float, help="Sample the test samples.")
+    return parser.parse_args()
+
+
+def eval_model(args: argparse.Namespace):
```

Comment:
might want od move this inside the package, then the script just does the high level things